model_templates/training/python3_pytorch/custom.py

```diff
@@ -49,9 +49,10 @@ def fit(
     X_train = subset_data(X)
     # Feel free to delete which ever one of these you aren't using
     if class_order:
-        estimator, optimizer, criterion = build_classifier(X_train)
+        estimator, optimizer, criterion = build_classifier(X_train, len(class_order))
+        print(estimator, optimizer, criterion)
```

Comment:
did you mean to leave this print statement?

Reply:
nope